addons/udes_stock/models/stock_move_line.py

```diff
@@ -16,6 +36,30 @@ class StockMoveLine(models.Model):
         readonly=True,
     )
 
+    @staticmethod
+    def _now_for_tracking_data():
+        """Override to change the tracking resolution"""
+        return datetime.now()
+
+    @api.model
+    def _add_user_tracking_data(self, vals):
+        """Inject/overwrite user tracking data into vals"""
+        vals["u_done_by_id"] = self.env.uid
+        vals["u_done_datetime"] = self._now_for_tracking_data()
+        return vals
+
+    @api.model
+    def create(self, vals):
+        """Extend to track the requester user"""
+        vals = self._add_user_tracking_data(vals)
```

Comment:
Does this mean the u_done_by on the move line is set on creation? When the qty_done is zero

Reply:
Does this have a potential to break something?

Reply:
qty_done > 0 i believe

Reply:
fixed